playbooks/common/openshift-cluster/upgrades/v3_1_to_v3_2/pre.yml

```diff
@@ -110,8 +110,8 @@
   # are modified to use the correct image tag.  However, this can trigger a
   # docker restart if new configuration is laid down which would immediately
   # pull the latest image and defeat the purpose of these tasks.
-  - openshift_cli
-  pre_tasks:
+  - openshift_docker
```

Comment:
I could just yank that feature, it would be nice, but maybe it's best if you have to re-run config playbook to add a new registry prior to upgrade. Then pre.yml can lookup latest version, it won't need to run the openshift_docker role anymore, or update systemd unit files.